tests/__init__.py

```diff
@@ -103,6 +103,10 @@ def test_addons_filtered(self):
                 ("bash", "-c", 'addons list -c | grep ,crm,'),
                 # absent_addon is missing and should fail
                 ("bash", "-c", "! addons list -px"),
+                # hr_payroll should have been filtered out
+                ("bash", "-c", "test ! -d auto/addons/hr_payroll"),
+                ("test", "-d", "auto/addons/hr"),
+                ("test", "-d", "auto/addons/hr_payroll_account"),
```

Comment:
Please use a glob in the test and inverse this test (adding `"!", `)

Reply:
what do you mean, I added three tests (don't know if the syntax is correct though) one to test that hr_payroll is filtered out and 2 more for testing that hr and hr_payroll_account still remaing there. Becasuse if you use the valid glob syntax `- "[!hr_payroll]" it will filter out all of them.